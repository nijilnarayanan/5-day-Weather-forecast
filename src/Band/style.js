import styled from 'styled-components'

export const Wrapper = styled.section`
	position: fixed;
	bottom: 0;
	max-width: 790px;
	margin: 0 auto;
	width: 100%;
	padding: 8px;
	@media screen and (min-width: 767px) {
		padding: 4px;
	}
`

export const Footer = styled.div`
	color: #e5e5e5;
	text-align: center;
	font-size: 12px;
	@media screen and (min-width: 767px) {
		font-size: 16px;
	}
`
