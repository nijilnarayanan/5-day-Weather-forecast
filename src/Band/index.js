import React from 'react'
import { Wrapper, Footer } from './style'

const Band = () => {
	return (
		<Wrapper>
			<Footer>Made with ❤️ by Nijil Narayanan</Footer>
		</Wrapper>
	)
}

export default Band
