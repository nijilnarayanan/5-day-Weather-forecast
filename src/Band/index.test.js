import React from 'react'
import { mount } from 'enzyme'
import Band from './index'

test('Run Test <Band />', () => {
	const component = mount(<Band />)
	expect(component.debug()).toMatchSnapshot()
})
