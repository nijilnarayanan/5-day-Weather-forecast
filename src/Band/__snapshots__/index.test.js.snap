// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Run Test <Band /> 1`] = `
"<Band>
  <styled.section>
    <section className=\\"sc-bdfBwQ klkbry\\">
      <styled.div>
        <div className=\\"sc-gsTCUz cyshaC\\">
          Made with ❤️ by Nijil Narayanan
        </div>
      </styled.div>
    </section>
  </styled.section>
</Band>"
`;
